pkg/rawyaml/resources.go

```diff
@@ -21,7 +21,7 @@ func ToObjects(c *chart.Chart) (result []runtime.Object, _ error) {
 		}
 		objs, err := yaml.ToObjects(bytes.NewBuffer(resource.Data))
 		if err != nil {
-			return nil, err
```

Comment:
Thanks in advance for your patience while I familiarize myself with the Fleet codebase.

Is returning an error here helpful in some cases where malformed yaml is unexpected? 

Also, is it expected for processing of a kustomize workload to invoke a function in the rawyaml package?

Reply:
Thanks for looking at it. I am still debating on this change. The problem with that issue is that we are marshalling non-k8s manifest(kustomize.yaml) into k8s object and that raised an error. It should probably need to exclude the file or ignore the error.

Reply:
Since filename `kustomization.yaml` has special meaning in the gitops space, would it be appropriate to explicitly exclude it rather than ignore all erroneous yaml manifests?

Reply:
Yeah that's the alternative way. I will dicuss with @ibuildthecloud to see what's the best way to solve this